Source/Scene/Cesium3DTileset.js

```diff
@@ -0,0 +1,1821 @@
+         * @readonly
+         *
+         * @default false
+         *
+         * @see Cesium3DTileset#allTilesLoaded
+         */
+        tilesLoaded : {
+            get : function() {
+                return this._tilesLoaded;
+            }
+        },
+
+        /**
+         * The url to a tileset.json file or to a directory containing a tileset.json file.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {String}
+         * @readonly
+         */
+        url : {
+            get : function() {
+                return this._url;
+            }
+        },
+
+        /**
+         * The base path that non-absolute paths in tileset.json are relative to.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {String}
+         * @readonly
+         */
+        basePath : {
+            get : function() {
+                return this._basePath;
+            }
+        },
+
+        /**
+         * The style, defined using the
+         * {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling|3D Tiles Styling language},
+         * applied to each feature in the tileset.
+         * <p>
+         * Assign <code>undefined</code> to remove the style, which will restore the visual
+         * appearance of the tileset to its default when no style was applied.
+         * </p>
+         * <p>
+         * The style is applied to a tile before the {@link Cesium3DTileset#tileVisible}
+         * event is raised, so code in <code>tileVisible</code> can manually set a feature's
+         * properties (e.g. color and show) after the style is applied. When
+         * a new style is assigned any manually set properties are overwritten.
+         * </p>
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {Cesium3DTileStyle}
+         *
+         * @default undefined
+         *
+         * @example
+         * tileset.style = new Cesium.Cesium3DTileStyle({
+         *    color : {
+         *        conditions : [
+         *            ['${Height} >= 100', 'color("purple", 0.5)'],
+         *            ['${Height} >= 50', 'color("red")'],
+         *            ['true', 'color("blue")']
+         *        ]
+         *    },
+         *    show : '${Height} > 0',
+         *    meta : {
+         *        description : '"Building id ${id} has height ${Height}."'
+         *    }
+         * });
+         *
+         * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling|3D Tiles Styling language}
+         */
+        style : {
+            get : function() {
+                return this._styleEngine.style;
+            },
+            set : function(value) {
+                this._styleEngine.style = value;
+            }
+        },
+
+        /**
+         * The maximum screen space error used to drive level of detail refinement.  This value helps determine when a tile
```

Comment:
We we better explain exactly what `maximumScreenSpaceError` is?  I don't think a user will have any idea from this description.  It's the geometric error in the tile, correct?